force-app/main/default/lwc/picklist/picklist.js

```diff
@@ -76,6 +74,7 @@ export default class Picklist extends LightningElement {
         this.options.forEach(option => {
             if (option.value === event.target.name) {
                 option.isSelected = !option.isSelected;
+                option.isChecked = true;
```

Comment:
Shouldn't isChecked = opiton.isSelected here as well?

Reply:
I had it that way but then I thought we are checking on 75 that if the value is equal to what I selected so I set it to true. I can change it back to option.isSelected

Reply:
No I think actually we want them both to be set to true because you can't unselect a multi select option.

Reply:
Ignore my last comment, on days of the week we let them deselect an option. So yea it would be unchecked then I would imagine.